AUTHORS.rst

```diff
@@ -1,9 +1,9 @@
 Credits
 -------
 
-This utility was created at `dealertrack technologies`_
+This utility was created at `Dealertrack Technologies`_
```

Comment:
this actually should be lowercase according to our marketing department. unless you know any different information, I would leave as is.